src/resilience/exec_space/openMP/OpenMPResParallel.hpp

```diff
@@ -87,38 +107,84 @@ class ParallelFor< FunctorType
                  , Kokkos::RangePolicy< Traits... >
                  , KokkosResilience::ResOpenMP>{
  private:
-  using Policy    = Kokkos::RangePolicy<Traits...>;
-  using WorkTag   = typename Policy::work_tag;
+  using Policy       = Kokkos::RangePolicy<Traits...>;
+  using WorkTag      = typename Policy::work_tag;
+  using LaunchBounds = typename Policy::launch_bounds;
+  using Member       = typename Policy::member_type;
 
   const FunctorType &  m_functor;
   const Policy m_policy;
 
   ParallelFor() = delete ;
   ParallelFor & operator = ( const ParallelFor & ) = delete ;
 
-  using surrogate_policy = Kokkos::RangePolicy < Kokkos::OpenMP, WorkTag >;
+  using surrogate_policy = Kokkos::RangePolicy < Kokkos::OpenMP, WorkTag, LaunchBounds>;
+
+#ifdef KR_ENABLE_WRAPPER
+  auto MakeWrapper (int64_t work_size, int64_t offset, const FunctorType &m_functor_0, const FunctorType &m_functor_1) const{
```

Comment:
```suggestion
  auto make_wrapper(int64_t work_size, int64_t offset, const FunctorType &functor_copy_0, const FunctorType &functor_copy_1) const {
```

Reply:
Kokkos CamelCases classes (e.g. `Kokkos::Impl::DeepCopy`) but snake_case for functions (e.g. `Kokkos::deep_copy`)